pkg/asset/agent/manifests/clusterimageset.go

```diff
@@ -55,27 +55,30 @@ func (a *ClusterImageSet) Generate(dependencies asset.Parents) error {
 		return err
 	}
 
+	clusterImageSet := &hivev1.ClusterImageSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("openshift-%s", currentVersion),
```

Comment:
IIUC, it means that the unconfigured-ignition would be locked to this release image version. It could be problematic since the appliance supports specifying a specific OCP release version. Can we maybe make it configurable? I.e. allow to bypass the validation in 'validateReleaseVersion' func.

Reply:
> the appliance supports specifying a specific OCP release version

What? No. The agent installer does not and will not support that.
And it doesn't even make any sense. The release image is built in to the appliance, so of course it has to use that one.

Reply:
ok, I thought we need to supply the ClusterImageSet. If the the release image in ClusterImageSet should be exactly the one in binary, it can be just auto-generated for the unconfigured ignition. It means that we have to use the binary, just wondered if we can import the IgnitionBase asset.

Reply:
A recent update made the ClusterImageSet optional. If one isn't provided it will generate one inside the ignition using the installer's release version.